docs/frontend-system/architecture/00-index.md

```diff
@@ -6,46 +6,74 @@ sidebar_label: Overview
 description: The structure and architecture of the new Frontend System
 ---
 
-> **NOTE: The new frontend system is in alpha and is only supported by a small number of plugins.**
+> **NOTE: The new frontend system is in alpha and is only supported by a small number of plugins. If you want to use the
+> new plugin system, you must migrate your entire Backstage application or start a new application from scratch. We do not
+> yet recommend migrating any apps to the new system.**
 
-## Building Blocks
+# Building Blocks
```

Comment:
The `title` from the frontmatter at the top will be an `H1`, if we feel like there should be more than one then maybe we should break this page up?

Reply:
My goal here was to simply create some structure in the sections, so that it was clear how they were related, but I can flatten it all out if you like.